test/test_helper.rb

```diff
@@ -180,6 +180,16 @@ def network_tokenization_credit_card(number = '4242424242424242', options = {})
       Billing::NetworkTokenizationCreditCard.new(defaults)
     end
 
+    # this mock is only useful with unit tests, as cryptograms generated by an EMV terminal
+    # are specific to the target acquirer, so remote tests using this mock will fail.
+    def credit_card_with_icc_data(options = {})
+      defaults = {
```

Comment:
It kind of seems like this method:
1) Doesn't do anything with `options`
2) Has a var called `defaults` that is actually options and never changes.

Do you need options to be passable? Or can you remove the argument and rename `defaults`?